Player.gd

```diff
@@ -50,7 +50,13 @@ var spawn_location = Vector2()
 
 #How much damage enemies do to do player
 export var basic_enemy_damage = 15
-
+	
+func _process(delta):
```

Comment:
Is there any reason to add this to _process rather than _physics_process? It's maybe better to keep all logic in one place, or at least group related logic (i.e. flipping the player and flipping their health bar) together.

Reply:
Implemented a better solution (on MandyBranch) using signals

Reply:
Ok.